### PR TITLE
Dynamic Cassandra Data Dir

### PR DESCRIPTION
**Pull request by alexbarta**

### CODE REVIEW

File: cmd/cain.go [hunk 1]
```diff
@@ -94,21 +96,23 @@ func NewBackupCmd(out io.Writer) *cobra.Command {
 	f.StringVar(&b.dst, "dst", "", "destination to backup to. Example: s3://bucket/cassandra")
 	f.IntVarP(&b.parallel, "parallel", "p", 1, "number of files to copy in parallel. set this flag to 0 for full parallelism")
 	f.Float64VarP(&b.bufferSize, "buffer-size", "b", 6.75, "in memory buffer size (MB) to use for files copy (buffer per file)")
+	f.StringVarP(&b.cassandraDataDir, "cassandra-data-dir", "", utils.GetStringEnvVar("CAIN_CASSANDRA_DATA_DIR", "/var/lib/cassandra/data"), "cassandra data directory")
```
Comment (maorfr) [inline cmd/cain.go]: We should probably add a note to the usage that this argument can be set using the env var.
For example: 
`cassandra data directory. Overrides $CASSANDRA_DATA_DIR`

Comment (alexbarta) [reply cmd/cain.go]: Added commit :)

**Comment by maorfr** [comment]
closes #7

**Comment by maorfr** [comment]
looks great!

ready to merge on your side?

**Comment by alexbarta** [comment]
yes go ahead please :)

**Pull request merged**